definitions/digital_analytics_domain/product_v1/latest_event_per_user_last_72_hours.sqlx

```diff
@@ -0,0 +1,48 @@
+/*
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     https://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+config {
+  type: "view",
+  schema: functions.productSchema("ga4"),
+  description: "Latest event per user within the last 72 hours window",
+  tags: ['ga4']
+}
+
+js {
+  const event_type_column = 'event_type';
+  const timestamp_column = 'event_timestamp';
+  const group_by_columns = [
+      'user_id',
```

Comment:
Is it the correct grouping of these records? If that's correct, LGTM.

Reply:
yes, user_id is the unique customer identifier that the customer need to set, if it's not set we'll fallback to user_pseudo_id.